src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -132,6 +132,7 @@
     <Compile Include="$(PathToMsalSources)\**\*.cs" Exclude="$(PathToMsalSources)\obj\**\*.*" />
     <Compile Remove="$(PathToMsalSources)\Platforms\**\*.*;$(PathToMsalSources)\Resources\*.cs" />
     <Compile Remove="$(PathToMsalSources)\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs" />
+    <Compile Include="Cache\CacheMissReason.cs" />
```

Comment:
I think this can be removed?

Reply:
Seems not to work without this. There is a configuration somewhere I am not aware of